.circleci/config.yml

```diff
@@ -108,298 +95,82 @@ jobs:
-      - run:
-          name: 'Test lit-protocol-cipher'
-          command: 'yarn workspace @requestnetwork/lit-protocol-cipher run test --ci --maxWorkers=1'
+          path: packages/thegraph-data-access/reports/
       - store_test_results:
-          path: packages/lit-protocol-cipher/reports/
-  test-epk-decryption:
-    docker:
-      - *node_image
-    working_directory: *working_directory
-    steps:
-      - attach_workspace:
-          at: *working_directory
-      - run:
-          name: 'Build epk-decryption'
-          command: 'yarn workspace @requestnetwork/epk-decryption run build'
-      - run:
-          name: 'Test epk-decryption'
-          command: 'yarn workspace @requestnetwork/epk-decryption run test --ci --maxWorkers=1'
+          path: packages/currency/reports/
       - store_test_results:
-          path: packages/epk-decryption/reports/
-  test-web3-signature:
-    docker:
-      - *node_image
-    working_directory: *working_directory
-    steps:
-      - attach_workspace:
-          at: *working_directory
-      - run:
-          name: 'Build web3-signature'
-          command: 'yarn workspace @requestnetwork/web3-signature run build'
-      - run:
-          name: 'Test web3-signature'
-          command: 'yarn workspace @requestnetwork/web3-signature run test --ci --maxWorkers=1'
+          path: packages/utils/reports/
       - store_test_results:
-          path: packages/web3-signature/reports/
-  test-integration-test:
+          path: packages/multi-format/reports/
+  test-integration:
     docker:
       - *node_image
       - *ipfs_image
       - *ganache_image
       - *postgres_image
       - *graph_image
+    resource_class: xlarge
     working_directory: *working_directory
     steps:
       - attach_workspace:
           at: *working_directory
-      - run:
-          name: 'Build integration-test'
-          command: 'yarn workspace @requestnetwork/integration-test run build'
       - run:
           name: 'Deploy test contract from smart-contracts'
           command: 'yarn workspace @requestnetwork/smart-contracts run deploy'
       - run: *step_graph_deploy
       - run:
           name: 'Start request-node'
-          command: 'yarn workspace @requestnetwork/request-node run start'
+          command: 'yarn start:request-node'
           background: true
       - run: *step_wait_for_node
       - run:
-          name: 'Test integration-test (node)'
-          command: 'yarn workspace @requestnetwork/integration-test run test:node --ci --maxWorkers=1'
-      - run:
-          name: 'Test integration-test (layers)'
-          command: 'yarn workspace @requestnetwork/integration-test run test:layers --ci --maxWorkers=1'
+          name: 'Test all'
+          command: 'yarn test --since master --scope @requestnetwork/request-client.js --scope @requestnetwork/smart-contracts --scope @requestnetwork/payment-detection --scope @requestnetwork/payment-processor --scope @requestnetwork/integration-test'
```

Comment:
Can we run these tests in parallel with the same Ganache and Request Node instance? It is probably OK; I am just wondering if some tests were designed to be run in isolation 🤔

Reply:
Yes, you are right. It worked here because it was --since master. I separated not the tests which requires Ganache from those that requires Request Node.